packages/lab/src/file-drop-zone/FileDropZone.css

```diff
@@ -1,106 +1,47 @@
-  --fileDropZone-title-lineHeight: var(--salt-text-lineHeight);
-}
-
-.saltFileDropZone {
-  color: var(--saltFileDropZone-text-color, var(--fileDropZone-text-color));
+  color: var(--saltFileDropZone-text-color, var(--salt-text-primary-foreground));
   display: inline-flex;
-  background: var(--saltFileDropZone-background, var(--fileDropZone-background));
+  background: var(--saltFileDropZone-background, var(--salt-container-primary-background));
   text-align: center;
   align-items: stretch;
   justify-content: center;
-  border-color: var(--saltFileDropZone-borderColor, var(--fileDropZone-borderColor));
-  border-style: var(--saltFileDropZone-borderStyle, var(--fileDropZone-borderStyle));
-  border-width: var(--saltFileDropZone-borderWidth, var(--fileDropZone-borderWidth));
+  border: var(--saltFileDropZone-border, var(--salt-size-border-strong) var(--salt-target-borderStyle) var(--salt-container-primary-borderColor));
   flex-direction: column;
-  padding: var(--saltFileDropZone-padding, var(--fileDropZone-padding));
-}
-
-.saltFileDropZone *:not(.saltFileDropZone-inputRoot, .saltFileDropZone-inputRoot *) {
-  pointer-events: none;
-}
-
-/* Styles applied to icon */
-.saltFileDropZone-icon {
-  fill: var(--fileDropZone-icon-color);
-  margin-bottom: var(--salt-size-unit);
-}
-
-.saltFileDropZone .saltFileDropZone-title:not(:last-child),
-.saltFileDropZone .saltFileDropZone-inputRoot:not(:last-child) {
-  margin-bottom: calc(var(--salt-size-unit) * 2);
+  padding: var(--salt-spacing-200);
+  gap: var(--salt-spacing-200);
+  width: 100%;
+  font-size: var(--saltFileDropZone-fontSize, var(--salt-text-fontSize));
+  font-family: var(--saltFileDropZone-fontFamily, var(--salt-text-fontFamily));
+  line-height: var(--saltFileDropZone-lineHeight, var(--salt-text-lineHeight));
 }
 
 /* Styles applied if `isActive={true}` */
-.saltFileDropZone-active:not(.saltFileDropZone-disabled) {
-  --fileDropZone-background: var(--salt-target-background-hover);
+.saltFileDropZone-active,
+.saltFileDropZone-active {
+  background: var(--saltFileDropZone-background-hover, var(--salt-target-background-hover));
+  border: var(--saltFileDropZone-border-hover, var(--salt-size-border-strong) var(--salt-target-borderStyle-hover) var(--salt-target-borderColor-hover));
 }
-
-/* Styles applied to description if `isRejected={true}`  */
-.saltFileDropZone-error .saltFileDropZone-description {
-  margin-bottom: calc(var(--salt-size-unit) * 2);
+.saltFileDropZone-active * {
+  pointer-events: none;
 }
-
-/* Styles applied if `isRejected={true}`  */
+/*!* Styles applied if `status="error" *!*/
 .saltFileDropZone-error {
-  --fileDropZone-background: var(--salt-status-error-background);
-  --fileDropZone-borderColor: var(--salt-status-error-borderColor);
-  --fileDropZone-icon-color: var(--salt-status-error-foreground);
+  border: var(--saltFileDropZone-error-border, var(--salt-size-border-strong) var(--salt-target-borderStyle) var(--salt-status-error-borderColor));
```

Comment:
I don't think we should be creating "variants" of variables. There should only be one, --saltFileDropZone-border

Reply:
Looks like this is still unresolved?